python_modules/libraries/dagster-gcp/dagster_gcp/gcs/sensor.py

```diff
@@ -5,7 +5,52 @@
 
 
 def get_gcs_keys(bucket, prefix="", since_key=None, gcs_session=None):
-    """Credits to @ajohnson5 for the original implementation of this method."""
+    """Return a list of updated keys in a GCS bucket.
+
+    Credits to @ajohnson5 for the original implementation of this method.
```

Comment:
This feels odd to have in the docs. Can we add it as a code comment or in the PR instead?

Reply:
I can, i did credit the author in the PR by linking to his and his name will also show up as co-author of the first commit.